analysis/demonhuntervengeance/src/modules/shadowlands/conduits/ShatteredRestoration.tsx

```diff
@@ -0,0 +1,63 @@
+import { formatNumber } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { HealEvent } from 'parser/core/Events';
+import { Options } from 'parser/core/EventSubscriber';
+import ConduitSpellText from 'parser/ui/ConduitSpellText';
+import ItemHealingDone from 'parser/ui/ItemHealingDone';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+
+export default class ShatteredRestoration extends Analyzer {
+  heal = 0;
+  factor = 0;
+  rank = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasConduitBySpellID(SPELLS.SHATTERED_RESTORATION.id);
+
+    if (!this.active) {
+      return;
+    }
+
+    const rank = this.selectedCombatant.conduitRankBySpellID(SPELLS.SHATTERED_RESTORATION.id);
+    this.rank = rank;
+    // rank 1 = 5%, scaling up to rank 11 atm at 10%
+    this.factor =
+      (this.selectedCombatant.likelyHasEmpoweredConduits() ? 0.06 : 0.05) + (rank - 1) * 0.005;
```

Comment:
If you want to, you could do something similiar to what has been done in analysis\hunterbeastmastery\src\constants.ts & analysis\hunterbeastmastery\src\modules\spells\conduits\Bloodletting.tsx using the generated spell scaling information relative to the rank rather than rely on calculating it yourself.